keyboards/adalyn/keymaps/vial/config.h

```diff
@@ -0,0 +1,21 @@
+ /* Copyright TJ Campie
```

Comment:
VIAL is not supported in the main repo, and the vial config and keymap should be removed.